app/models/consultation.rb

```diff
@@ -371,6 +379,14 @@ def needs_site_visit?
     planning_application.prior_approval? ? neighbour_responses.objection.any? : planning_application.application_type.site_visits?
   end
 
+  def neighbour_review
+    neighbour_reviews.order(:created_at).last
+  end
```

Comment:
This will always run an SQL query every time that it's called - depending on how you address the source association comment I made above you can either implement this as a `has_one` association that will cache the lookup or use the association extension method.